.github/workflows/CI.yml

```diff
@@ -0,0 +1,23 @@
+name: Check code format
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  build:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: install clang-format
+        run: |
+          sudo apt update
+          sudo apt install -y clang-format
+      - name: checkout repository content
+        uses: actions/checkout@v2
```

Comment:
**suggestion:** I think the latest version is [v3](https://github.com/actions/checkout) at the moment.

```suggestion
        uses: actions/checkout@v3
```